addons/sys_rack/stringtable.xml

```diff
@@ -127,5 +127,11 @@
             <English>D.Up</English>
             <Italian>C. Alto</Italian>
         </Key>
+        <Key ID="STR_ACRE_sys_rack_useAllRacks">
```

Comment:
The other one is actually "useRadio", I don't think it matters much, but consistency is good.